main.tf

```diff
@@ -31,13 +31,27 @@ resource "aws_security_group" "hello-terra-ssh-http" {
 }
 
 resource "aws_instance" "hello-terra" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure IMDv1 is disabled for AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances that have IMDv1 enabled are vulnerable to server side request forgery (SSRF) attacks.
 
**Remediation**
IMDv1 is vulnerable to SSRF attack. Thus, EC2 Instance should have metadata_options block with either http_endpoint attribute set to disabled or ensure http_tokens attribute is set to required. This ensures IMDv2 is being used.

source: accurics-stagetwo